webclient/pages/pf/_uuid/index.vue

```diff
@@ -159,7 +159,7 @@ export default {
         {
           target: '.el-icon-setting',
           content:
-            '<el-card class="box-card"><div slot="header" class="clearfix"><span>F1 gets you the dashboard</span></div><div><img src=https://cdn.osxdaily.com/wp-content/uploads/2011/01/quick-system-preferences-option-key-macbook.jpg height=300px class="image"></div></el-card>',
+            '<div class="el-card box-card is-always-shadow"><div class="el-card__header"><div class="clearfix"><span>F1 gets you the dashboard</span> </div></div><div class="el-card__body"> <div class="text item"><img src=https://cdn.osxdaily.com/wp-content/uploads/2011/01/quick-system-preferences-option-key-macbook.jpg height=300px class="image"></div></div></div>',
```

Comment:
The offical example at https://element.eleme.io/#/en-US/component/card implements it differently. Why have you done differently?